recipes-asteroid/asteroid-weather/asteroid-weather_git.bb

```diff
@@ -12,4 +12,4 @@ inherit cmake_qt5
 
 DEPENDS += "qml-asteroid asteroid-generate-desktop-native nemo-qml-plugin-configuration qttools-native qtdeclarative-native"
 RDEPENDS:${PN} += "nemo-qml-plugin-configuration"
-FILES:${PN} += "/usr/share/translations/"
+FILES:${PN} += "/usr/share/translations/ ${libdir}/asteroid-weather.so"
```

Comment:
fwiw, it could have been nice to use ${PN} rather than the recipe names to make these easier to share.

also it could have been good to extract the shared logic in a bbclass

but nothing bad, I'm also happy with the PR as is

Reply:
Good idea. Created an issue for that: https://github.com/AsteroidOS/meta-asteroid/issues/160